Setup/Product.wxs

```diff
@@ -599,14 +589,6 @@
               - The add-in feature state (!feature-state) for the Visual Studio version is installed (3 = INSTALLSTATE_LOCAL). See:
                 Conditional Statement Syntax (http://msdn.microsoft.com/en-us/library/aa368012.aspx)
             -->
-            <Custom Action="RemoveVS2005AddInCommands"
-                 After="RemoveFiles">Installed AND NOT UPGRADINGPRODUCTCODE AND VS2005INSTFOLDER
-                                     AND !VS2005FeatureId=3</Custom>
```

Comment:
> so if possible, it would be good to leave the old addin installed for 2005 and 2008.

But I don't know how they will remove old addin If they need. I think it's better that they continue to use the old versions of GitExt if they want to use it with VS2005,2008